BetaCameras/RealSense2/RealSense2.cs

```diff
@@ -1200,17 +1024,8 @@ RangeParamDesc<int> FrameQueueSizeColorDesc
 
                 if(this.IsConnected)
                 {
-                    RealSense2API.QueryOptionInfo(
-                    _pipeline,
-                    RealSense2API.SensorName.COLOR,
-                    RealSense2API.Option.FRAMES_QUEUE_SIZE,
-                    out float min,
-                    out float max,
-                    out float step,
-                    out float def,
-                    out string desc);
-
-                    res = new RangeParamDesc<int>((int)min, (int)max);
+                    var option = QueryOption(RealSense2API.Option.FRAMES_QUEUE_SIZE, RealSense2API.SensorName.COLOR);
```

Comment:
I would even write a method which contains these two lines, `QueryOption` and `new RangeParamDesc`, just because I wouldn't want to write it over and over again (but it's there already, so you may as well keep it).